tasks/src/app/simple-button/simple-button.component.html

```diff
@@ -1 +1,3 @@
-<button (click)="increseNumberOfCounts()">Number of clicks {{numberOfClicks}}</button>
\ No newline at end of file
+<button (click)="changeInputStatus()">{{ inputStatus?("Disable"):("Enable") }}</button><input type="text"
+	placeholder="Name" [disabled]="!inputStatus" (keyup)="handleKeyUp($event)">
+<p ng-hide="!inputStatus">{{ showedText }}</p>
```

Comment:
```suggestion
<p *ngIf="!inputStatus">{{ showedText }}</p>
```
ng-hide is from AngularJs (the first version of Angular which is not maintained anymore)
use *ngIf directive instead